apps/backend/src/libs/types/repository.type.ts

```diff
@@ -1,9 +1,9 @@
 type Repository<T = unknown> = {
 	create(payload: unknown): Promise<T>;
-	delete(): Promise<boolean>;
-	find(id: number): Promise<T>;
+	delete(id: number): Promise<boolean>;
+	find(payload: unknown): Promise<T>;
```

Comment:
`find` can only receive `id`

Reply:
Maybe rename to findById? I had an issue where linter thought that argument that i pass to find method on user service is function (the argument was of type number in fact). Renaming the method to findById helped.

Reply:
@ihorLysak , yeah, I had the same issue yesterday with eslint, if I am not mistaken, saying something like "find() method should be used not just for id" when was calling repository.find in service.find for `categories` module.
But just a minute ago I've pushed commit where I've changed find params to have (id: number) only and everything works fine. Maybe the issue was that I was using different method instead of `this.<someModel>.query().findById(id);` in repository
Try this out, maybe you have the same issue

Final code (without errors):
```
// category.repository.ts
public async find(id: number): Promise<CategoryEntity | null> {
		const category = await this.categoryModel.query().findById(id);

		return category
			? CategoryEntity.initialize({
					categoryId: category.categoryId,
					createdAt: category.createdAt,
					id: category.id,
					name: category.name,
					score: category.score,
					updatedAt: category.updatedAt,
					userId: category.userId,
				})
			: null;
	}
```